scripts/psycorona_modelling.R

```diff
@@ -0,0 +1,36 @@
+rm(list = ls())
```

Comment:
Please never use this. It is considered antisocial behavior to remove all objects from the global environment